integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.4.21
```

Comment:
why this change?

Reply:
Not needed anymore since I decided to avoid modifying the Docker file.